src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -381,29 +381,12 @@ private static GpioDriver GetBestDriverForBoard()
         /// <returns>A driver that works with the board the program is executing on.</returns>
         private static GpioDriver GetBestDriverForBoardOnLinux()
         {
-            string[] cpuInfoLines = File.ReadAllLines(CpuInfoPath);
-            Regex regex = new Regex(@"Hardware\s*:\s*(.*)");
-            foreach (string cpuInfoLine in cpuInfoLines)
+            try
+            {
+                return new RaspberryPi3Driver();
+            }
+            catch (PlatformNotSupportedException)
             {
```

Comment:
So you're catching but don't do anything here? Maybe just move the line `return UnixDriver.Create();` here